lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -227,6 +232,7 @@ if (require.main === module) {
   console.log(`offline: listening on http://localhost:${offlinePort}`);
```

Comment:
one day we should split this off into a `smoke-static-server-entry.js` or something ..